plugins/woocommerce-admin/client/core-profiler/index.tsx

```diff
@@ -100,6 +110,65 @@ const Extensions = ( {
 	);
 };
 
+const getAllowTrackingOption = async () =>
```

Comment:
Do you think we should move these out into more specific collections instead of being located in this file?

Reply:
Yeah, that would be great!